.github/workflows/ci-complete.yml

```diff
@@ -78,26 +73,35 @@ jobs:
         uses: ./.github/actions/gh-api-update-status
         with:
           gh-token: ${{ secrets.GITHUB_TOKEN }}
+          repository: ${{ github.repository }}
           commit_sha: ${{ github.event.workflow_run.head_sha }}
           url: '${{ github.event.repository.html_url }}/actions/runs/${{ github.run_id }}'
```

Comment:
just an idea. We need to check the CI workflow to find out the root cause for nonexistent build scan, so `url` can be changed to `${{ github.event.workflow_run.html_url }}` which references to "CI" workflow. WDYT?